Add documentation for Playbooks (#89)
flanksource/docs: docs/playbooks/quick-start.md

# Quick Start

In this walkthrough, we'll create and run a playbook that scales a Kubernetes deployment.
The only prerequisites are

- you have a config item for a Kubernetes Deployment
- you have kubectl configured.

## 1. Create the playbook

> ```yaml

Reviewer: ```suggestion
```yaml title="scale-deployment.yaml"
```

apiVersion: mission-control.flanksource.com/v1
kind: Playbook
metadata:
  name: scale-deployment
spec:
  description: Scale deployment
  configs:
    - type: Kubernetes::Deployment
      tags:
        namespace: default
        cluster: local-kind-cluster
  parameters:
    - name: replicas
      label: The new desired number of replicas.
  actions:
    - name: 'scale deployment'
      exec:
        script: kubectl scale --replicas={{.params.replicas}} deployment {{.config.name}}
```

> This playbook is designed to run a configs of type `Kubernetes::Deployment` & having labels namespace: `default` & cluster: `local-kind-cluster`.

Reviewer: ```suggestion
This playbook is designed to run on a Kubernetes Depolyment (with a config type of `Kubernetes::Deployment`)
```

If you need, you can adjust the tags to your needs.

Save the above YAML to a file called `scale-deployment.yaml`. Then, to store the playbook, run:

```bash
kubectl apply -f scale-deployment.yaml
```

Please check the `playbooks` table in the database to see if the playbook has been created.

## 2. Run the playbook

To run the playbook, we need its id. In addition to that please get the id for the deployment config item that you want to scale.

> ```bash

Reviewer: Rather use CRD definition to apply

Author: Wdym?
